blockchain/integrity/integrity_test.go

```diff
@@ -692,7 +693,7 @@ func addTestingGetBlockHash(t *testing.T, g genesis.Genesis, bc blockchain.Block
 			bcHash, err = dao.GetBlockHash(targetHeight)
 			require.NoError(err)
 		}
-		require.Equal(r.Logs()[0].Topics[0], bcHash)
+		require.Equal(r.Logs()[0].Topics[1], bcHash)
```

Comment:
`topic[0]` is actually the contract address and wrong value to compare, the correct value is `topic[1]`
so test passes b/c before Aleutian height there's a bug that all topics are overwritten by last topic's value so `topic[0] == topic[1]` in this test
so set the Aleutian height to correct the test (test will fail if still compare with topic[0])